src/main/java/com/couchbase/lite/store/SQLiteStore.java

```diff
@@ -2315,7 +2339,7 @@ protected int pruneRevsToMaxDepth(int maxDepth) throws CouchbaseLiteException {
         try {
             beginTransaction();
             for (Long docNumericID : toPrune.keySet())
-                outPruned += pruneDocument(docNumericID, toPrune.get(docNumericID).intValue() + 1);
+                outPruned += pruneDocument("?", docNumericID, toPrune.get(docNumericID).intValue() + 1);
```

Comment:
To be consistent with iOS, CBL Java can keep docID param. But it might not be used.
Do you have suggestion?